tests/console/yast2_ntpclient.pm

```diff
@@ -34,7 +34,8 @@ sub run {
     # test often fails due to info kernel messages disrupting screen
     # decrease logging level to warning to avoid this
     assert_script_run 'dmesg -n 4';
-
+    record_soft_failure 'bsc#1011815';
+    
```

Comment:
Adding a blank line with whitespace at EOL. I don't like it … but the important part is, that *tidy* doesn't allow it either ;P